docs/source/tutorial/type_check.rst

```diff
@@ -281,9 +280,9 @@ And then, get each type:
 
    x_type, y_type = in_types
 
-Don't get each value before check ``in_types.size()``.
-When the number of argument is illegal, this process may fail.
-For example, this code doesn't work when the size of ``in_types`` is zero:
+Don't get each value before checking ``in_types.size()``.
+When the number of argument is illegal, this will fail.
```

Comment:
This change alter the meaning of the document. I am now investigating this change is acceptable.